robotics-ai-suite/robot-vision-control/docs/source/getstarted/camera_setup_gmsl.rst

```diff
@@ -0,0 +1,3848 @@
+.. _enableGmsl:
+
+Camera GMSL setup
+##############################################################
+
+The Gigabit Multimedia Serial Link™ (GMSL) technology for cameras modules offers lower power consumption, reduced ESD/EMI noise, lower latency, and longer cable length compared to USB, while providing higher bandwidth compared to Ethernet.
+
+Third-party MIPI Camera vendors, including  `D3 Embedded® <https://www.d3embedded.com/>`_ and `oToBrite® <https://www.otobrite.com/>`_ , offers a rich products line of GMSL2 Camera modules for every solution need. The `RealSense™ Depth Camera D457 product <https://realsenseai.com/products/d457-gmsl-fakra/>`_  adds GMSL serializer and FAKRA connector to their camera modules product line.
+
+GMSL uses very popular `SerDes` technique across telecom, industrial, and cable interconnect applications to meet the growing demand of high data rates, long distance support, and better performance.
+This serial link technology also performs reliably in the harsh industrial and outside environments to deliver data fast with low latency, providing data transmission over a single coaxial cable or differential pair cables (STP, SPP, etc.) to minimize the number of Input/Output pins and interconnects.
+
+A GMSL product design based |Intel| Core™ Ultra Series 1 and 2 (Arrow Lake-U/H) or 12th/13th/14th Gen |Intel| Core™ products can be illustrated as followed :
+
+   .. figure:: images/gmsl/GMSL-overview.png
+      :align: center
+
+   - The **GMSL2 Camera modules**, designed by 3rd Party GMSL2 Camera vendors, combine one or more Camera Sensors and a GMSL2 *Serializer* (ex. MAX9295)
+   - The **Add-in-Card (AIC)**, designed by either ODM/OEMs or 3rd Party GMSL2 Camera vendors, provide multiple GMSL2 *Derializer* (ex. MAX9296A)
+   - The **|Intel|-based Motherboad**, designed by ODM/OEMs, provide Mobile Industry Processor Interface (MIPI) Camera Serial Interface (CSI) interface exposed by |Intel| Core™ Ultra Series 1 and 2 (Arrow Lake-U/H) and 12th/13th/14th Gen |Intel| Core™ products.
```

Comment:
The open-source documentation does not use substitutions such as `|Intel|`, or `|ECI|`.  These need to be replaced.  When referring to the project `OEP` should be used, not `ECI`.  It is OK to refer to the `ECI APT repository`, but `ECI` in general should not be used.

Reply:
ok. will do